packages/react/src/primitives/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { ComponentClassName, isFunction } from '@aws-amplify/ui';
+import { BaseAccordionProps, AccordionProps } from './types';
+import { ForwardRefPrimitive, Primitive } from '../types/view';
+import { View } from '../View';
+import { AccordionContext } from './AccordionContext';
+import { AccordionItem } from './AccordionItem';
+
+const AccordionPrimitive: Primitive<AccordionProps, 'div'> = (
+  {
+    children,
+    className,
+    defaultValue,
+    allowMultiple,
+    allowToggle,
```

Comment:
Yea I was thinking that, I kinda wanted the boolean props to be false by default so you can just add them if you want them rather than turn them off, but didn't know what the opposite prop name for that would be...